Mailbox.Tests/ProgramTests.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.TestPlatform.TestHost;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Mailbox.Tests
+{
+    [TestClass]
+    public class ProgramTests
+    {
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void AddMailbox_nullMailboxes()
+        {
+            Program.AddNewMailbox(null, "Jim", "Lakes", Size.Small);
+        }
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void AddMailbox_nullFirstName()
+        {
+            Mailboxes mailboxes = new Mailboxes(getTestData(), 50, 10);
+            Program.AddNewMailbox(mailboxes, null, "Lakes", Size.Small);
+        }
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void AddMailbox_nullLastName()
+        {
+            Mailboxes mailboxes = new Mailboxes(getTestData(), 50, 10);
+            Program.AddNewMailbox(null, "Jim", null, Size.Small);
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void getMailBoxDetails_NullMailboxes()
```

Comment:
Method names should be CamelCase
```suggestion
        public void GetMailBoxDetails_NullMailboxes()
```